apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/R4PatientResourceProvider.java

```diff
@@ -224,6 +227,9 @@ public Bundle searchByLogicalId(
       patients = Collections.emptyList();
     } else {
       try {
+        // Add bene_id to MDC logs
+        LoggingUtils.logBeneIdToMdc(Long.parseLong(logicalId.getValue()));
```

Comment:
same comment about the Long parsing. A utility method is probably the way to go

Reply:
this one might be fine actually since we have to be able to parse the long above for this to work at all. I think that anywhere that we are parsing the long ONLY to write the MDC is where we want to be sure to try/catch and not let that logic cause a failure that might otherwise have been fine.

Reply:
Got it.